test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6266,7 +6295,7 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
             return None
 
         if submit_dir is None:
-            submit_dir = tempfile.gettempdir()
+            submit_dir = pwd.getpwnam(obj.username)[5]
```

Comment:
This is desired for Cray, but as per Hiren's comment above he has suggested to use user's home directory for non-cray too.